### PR TITLE
258 - updates for jupyter notebooks page

### PR DESCRIPTION
**Pull request by mxochicale**
# Addressing #258

### CODE REVIEW

File: docs/pages/jupyter-notebooks.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ We generally recommend packaging reusable code components into Python modules wh
 However occasionally we support researchers who prefer a notebook environment or projects which want to provide examples and tutorials in notebook format.
 Notebooks can also be a valid alternative to Python scripts for running and recording the results of numerical experiments for example.
 
+## Live executable environment
```
Comment (samcunliffe) [inline docs/pages/jupyter-notebooks.md]:
```suggestion
## Live executable environments
```

File: docs/pages/jupyter-notebooks.md [hunk 2]
```diff
@@ -9,6 +9,14 @@ We generally recommend packaging reusable code components into Python modules wh
...
+| [binder](https://mybinder.org/)                                                                                                                                                                      | Turns a Git repository into a collection of interactive notebooks                                                                                             | 🟢  | [LiPyphilic](https://github.com/p-j-smith/lipyphilic) |
+| [Google Colabs](https://colab.google/)                                                                                                                                                               | Colab is a hosted Jupyter Notebook service that requires no setup to use and provides free access to computing resources, including GPUs and TPUs.            | 🟢  |                                                       |
```
Comment (samcunliffe) [inline docs/pages/jupyter-notebooks.md]:
```suggestion
| [Google Colab](https://colab.google/)                                                                                                                                                               | Colab is a hosted Jupyter Notebook service that requires no setup to use and provides free access to computing resources, including GPUs and TPUs.            | 🟢  |                                                       |
```

**Review by samcunliffe — APPROVED** [review]
🫐

File: docs/pages/jupyter-notebooks.md [hunk 3]
```diff
@@ -9,6 +9,14 @@ We generally recommend packaging reusable code components into Python modules wh
...
+| [Google Colab](https://colab.google/)                                                                                                                                                                | Colab is a hosted Jupyter Notebook service that requires no setup to use and provides free access to computing resources, including GPUs and TPUs.            | 🟢  |                                                       |
+| [GitHub codespace with JupyterLab](https://docs.github.com/en/codespaces/developing-in-a-codespace/getting-started-with-github-codespaces-for-machine-learning#opening-your-codespace-in-jupyterlab) | A codespace is a development environment that is hosted in the cloud and can run [Python and Jupyter notebooks](https://github.com/github/codespaces-jupyter) | 🟠  |
```
Comment (dstansby) [inline docs/pages/jupyter-notebooks.md:18]:
```suggestion
| [GitHub codespaces with JupyterLab](https://docs.github.com/en/codespaces/developing-in-a-codespace/getting-started-with-github-codespaces-for-machine-learning#opening-your-codespace-in-jupyterlab) | A codespace is a development environment that is hosted in the cloud and can run [Python and Jupyter notebooks](https://github.com/github/codespaces-jupyter) | 🟠  |
```

**Review by dstansby — APPROVED** [review]

**Pull request merged**